### DISCUSSION TITLE
language not found

### DISCUSSION THREAD

**Discussion by zxhTom** [AGou-ops/dotfiles#14] [Q&A]
Spawning language server with cmd: `vim-language-server` failed. The language server is either not installed, missing from PATH, or not executable

i found export PATH=~/.local/share/nvim/lsp_servers/bin. you configure

But I found that the real install path is configured with ~/.local/share/lsp_servers

why

**Comment by AGou-ops** [comment]
Make sure that the command can be executed directly in the terminal. It depends on your `mason` configuration.

like this:

<img width="1546" alt="image" src="https://user-images.githubusercontent.com/57939102/212693260-4023e76b-a509-4e69-ac19-8f9ce5f8177f.png">

You can change it to your own.

**Comment by zxhTom** [comment]
邮件已收到！谢谢配合。

**Comment by zxhTom** [comment]
![2023-01-17_08-22](https://user-images.githubusercontent.com/12728104/212784039-11e875e8-0f3c-4fb0-b94b-3954e579dedd.png)

Why is the structure of my installation different from yours? I installed it through lsp-installer

**Comment by AGou-ops** [comment]
> ![2023-01-17_08-22](https://user-images.githubusercontent.com/12728104/212784039-11e875e8-0f3c-4fb0-b94b-3954e579dedd.png)
> 
> Why is the structure of my installation different from yours? I installed it through lsp-installer

Idk, I used `mason`.
